cmd/mesh-bootstrap/main.go

```diff
@@ -141,11 +144,15 @@ func main() {
 		libp2p.ListenAddrs(bindAddrs...),
 		libp2p.Identity(privKey),
 		libp2p.ConnectionManager(connManager),
-		libp2p.EnableRelay(circuit.OptHop),
 		libp2p.EnableAutoRelay(),
 		libp2p.Routing(newDHT),
 		libp2p.AddrsFactory(newAddrsFactory(advertiseAddrs)),
 	}
+	if config.EnableRelayHost {
+		opts = append(opts, libp2p.EnableRelay(circuit.OptHop))
+	} else {
+		opts = append(opts, libp2p.EnableRelay())
```

Comment:
Looks like either way, we're enabling relay? Or is it not actually enabled if no args are passed in?

Reply:
The name of this configuration option in libp2p is confusing. Here's the documentation for it:

> // EnableRelay configures libp2p to enable the relay transport with
> // configuration options. By default, this option only configures libp2p to
> // accept inbound connections from relays and make outbound connections
> // _through_ relays when requested by the remote peer. (default: enabled)
> //
> // To _act_ as a relay, pass the circuit.OptHop option.

Reply:
What we care about here is whether the bootstrap node can _act_ as a relay, and that is determined by the presence of the `circuit.OptHop` option.

Reply:
Makes sense, thanks for clarifying.